src/workerd/server/tests/python/py_wd_test.bzl

```diff
@@ -13,12 +13,14 @@ def py_wd_test(
         data = None,
         name = None,
         python_flags = "all",
+        skip_python_flags = [],
         args = [],
         size = "enormous",
         tags = [],
         **kwargs):
     if python_flags == "all":
         python_flags = FEATURE_FLAGS.keys()
+    python_flags = [flag for flag in python_flags if flag not in skip_python_flags]
```

Comment:
Nit:

Something like this might be more readable:

Python_flags.difference(skip_python_flags)

Reply:
is `list.difference` a thing in starlark?

Reply:
It seems it's only available on sets, not lists. My bad.

Reply:
But I don't think starlark has sets at all. I tried to do `python_flags = set(python_flags).difference(skip_python_flags)` like I'd do in Python but I get `ERROR: name 'set' is not defined`

Reply:
It seems - is the correct operator

> The - operation on two sets returns the difference of the two sets: a set containing the elements found in the left-hand side set but not the right-hand side set.

set([1, 2]) - set([2, 3])  # set([1])
set([1, 2]) - set([3, 4])  # set([1, 2])

Reply:
But that's still Python, right? Not starlark?

Reply:
It is Starlark - https://github.com/bazelbuild/starlark/blob/master/spec.md#sets

Reply:
Hmm... looks like sets are a fairly recent addition that maybe haven't made it into that part of the docs yet: https://github.com/bazelbuild/starlark/pull/290